docs/UserGuide.md

```diff
@@ -105,17 +105,25 @@ The `Tool Bar` at the top provides a set of configuration options that control t
   * Multiple keywords/terms can be used, separated by spaces.
   * Entries that contain _any_ (not necessarily _all_) of the search terms will be displayed.
   * The keywords used to filter author and repository are case-insensitive.
-* `Sort by` : sorting criteria for the Chart Panel
-  * `Total Contribution` : the amount of lines, written by the author, in the repository.
-  * `Variance` : the [variance](https://en.wikipedia.org/wiki/Variance) of the number of lines that the author has contributed to the repository daily, sorted from low to high.
-  * `Author Name` : the author's display name.
-  * `Repo/Branch Name` : see note [1] below.
+* `Group by` : grouping criteria for the rows of results
+  * `None` : results will not be grouped in any particular way.
+  * `Repo/Branch` : results will be grouped by repositories and its' associating branches.
+  * `Author` : results will be grouped by the name of the author. Contributions made to multiple repositories by a particular author will be grouped under the author.
+* `Sort groups by`: sorting criteria for the main group
+  * `Name` : groups will be sorted by GitHub ID in alphabetical order.
+  * `Repo/branch` : groups will be sorted in alphabetical order by the name of the repo, followed by name of the branch. See note [1] below.
+  * `Contribution` : groups will be sorted by the combined contributions within a group, in the order of number of lines added
+  * `Variance` : the [variance](https://en.wikipedia.org/wiki/Variance) of the number of lines that the group has contributed to the repository daily.
+* `Sort within groups by`: sorting criteria within each group
+  * `Name` : sorted by GitHub ID in alphabetical order.
+  * `Repo/branch` : sorted in alphabetical order by the name of the repo, followed by name of the branch. See note [1] below.
+  * `Contribution` : sorted by individual contributions in the order of number of lines added
+  * `Variance` : the [variance](https://en.wikipedia.org/wiki/Variance) of the number of lines that the group has contributed to the repository daily.
```

Comment:
It seems like the description for `variance` follows a very different style from the other 3. Also, the description is the same as the  `Sort groups by`.

The description for the fields under the `Sort within groups by` also follows a very different style from the other descriptions.

Try to maintain the same style, either blends into the existing style or standardize all the description. Having too many different styles in a same paragraph can easily confuse the users.

Reply:
I re-wrote the description for variance part as I believe the previous version isn't well defined. Do let me know if the description needs to be improved further.